Makefile.in

```diff
@@ -623,6 +623,9 @@ endif
 	@for f in project/cmake/scripts/common/*.cmake; do \
 	  install -m 0644 $$f $(DESTDIR)$(libdir)/@APP_NAME_LC@; \
 	done
+ifeq ($(findstring linux,@ARCH@),linux)
```

Comment:
same reason as why @Montellese wanted it limited to linux in the first place: its only used on linux and we don't want to clutter other platforms.